app/src/main/java/org/mozilla/tv/firefox/pocket/PocketVideoFetchScheduler.kt

```diff
@@ -27,6 +27,8 @@ private const val FETCH_UNIQUE_WORK_NAME = "PocketFetch"
 class PocketVideoFetchScheduler(
     private val isPocketEnabledByLocale: () -> Boolean
 ) : LifecycleObserver {
+    private var qaFetchDelayOverrideMillis: Long? = null
+    private var isQABuild = false
```

Comment:
nit: I don't love that these initial states are duplicated with the reset values in `resetQAFetchDelayOverrides`. However, most of the solutions I can think of have a lot of boilerplate or are misleading. Maybe:
```kotlin
    private var qaFetchDelayOverrideMillis: Long? = null
    private var isQABuild = false
    init { resetQAFetchDelayOverrides() } // replaces initial values above to avoid duplication.
```

What do you think?